### PR TITLE
[Kotlin Migration] StringUtilTest

### PR DESCRIPTION
**Pull request by BrayanDSO**
## Purpose / Description
I wanted to test #10779 on my Windows PC.

## Approach
1. Converted file to Kotlin
2. Solved errors
3. Triggered `tools\migrate.sh`

I ran `migrate.sh` with Git BASH. The script worked perfectly without requiring any changes. Very, very, good.

Git version: 2.35.1.2
Windows 10 21H2

## How Has This Been Tested?

Ran the tests

## Checklist
_Please, go through these checks before submitting the PR._

- [X] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [X] You have a descriptive commit message with a short title (first line, max 50 chars).
- [X] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [X] You have commented your code, particularly in hard-to-understand areas
- [X] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

**Comment by david-allison** [comment]
```
/home/runner/work/Anki-Android/Anki-Android/AnkiDroid/src/test/java/com/ichi2/utils/StringUtilTest.kt:54: Error: Use kotlin.test.junit.JUnitAsserter instead of JUnit in Kotlin [LegacyNullAssertionDetector from com.ichi2.anki.lint]
        assertNull(strip(null))
        ~~~~~~~~~~~~~~~~~~~~~~~
1 errors, 0 warnings

```

**Review by david-allison — CHANGES_REQUESTED** [review]
One lint fix, the rest is fine

**Review by david-allison — APPROVED** [review]
LGTM

File: AnkiDroid/kotlinMigration.gradle [hunk 1]
```diff
@@ -42,8 +42,8 @@ permission notice:
 
 // Example of class name: "/com/ichi2/anki/UIUtils.kt"
 // Ensure that it starts with '/' (slash)
-def source = Source.MAIN
```
Comment (mikehardy) [inline AnkiDroid/kotlinMigration.gradle:45]: Hey it does the MAIN / TEST detection, nice @Arthur-Milchior !

**Review by mikehardy — APPROVED** [review]
Nice!

**Pull request merged**